### PR TITLE
WIP: Create readme

### PR DESCRIPTION
**Pull request by filipblnt**
see https://github.com/blindnet-io/devrel-management/issues/9 & https://github.com/blindnet-io/openness-framework/pull/33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  <a href="https://github.com/blindnet-io/{project-short-name}/issues">Submit an Issue</a>
+  &nbsp;•&nbsp;
+  <a href="https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg">Online Chat</a>
+  <br>
+  <br>
+</p>
+
+## About
+
+This is the .Net Framework [server-side SDK](https://docs.blindnet.io/docs/extra/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```
Comment (m4rk055) [inline README.md:29]:
```suggestion
This is the .Net Framework [server-side SDK](https://docs.blindnet.io/docs/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+- Generate [authentication tokens](https://docs.blindnet.io/docs/extra/glossary#authentication-token) to allow users to access blindnet.
```
Comment (m4rk055) [inline README.md:31]:
```suggestion
- Generate [authentication tokens](https://docs.blindnet.io/docs/glossary#authentication-token) to allow users to access blindnet.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.
+
+## Installation
+
+<!-- FIXME: add install steps -->
+Use [{package manager or plateform}][install-tool] to install {project's name}:
+
+```bash
+{install-command}
+```
```
Comment (noelmace) [inline README.md:45]: TODO @blindnet-io/engineering

File: README.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+<!-- FIXME: add API Reference
+## Usage
+
+📑 The API reference of blindnet devkit Node.js Server SDK is available on [blindnet.dev](https://docs.blindnet.io/docs/api_reference/server/node.js/latest).
+-->
```
Comment (noelmace) [inline README.md:51]: TODO @blindnet-io/engineering

**Review by noelmace — COMMENTED** [review]
TODO List for @blindnet-io/engineering before merging this in order to close https://github.com/blindnet-io/devrel-management/issues/9

**Pull request merged**